.github/workflows/lint.yml

```diff
@@ -0,0 +1,100 @@
+name: lint
+
+on: [push, pull_request]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
Is there any reason to use 3.8 and not 3.10?

Reply:
3.8 is our targeted minimum version

Reply:
Oh sorry I'm dumb
It makes sense to me now